### DISCUSSION TITLE
v7.2.1 Release of PowerShell

### DISCUSSION THREAD

**Discussion by anmenaga** [PowerShell/PowerShell#16614] [General]
### General Cmdlet Updates and Fixes

- Remove declaration of experimental features in Utility module manifest as they are stable (#16460)
- Bring back pwsh.exe for framework dependent packages to support Start-Job (#16535)
- Change default for `$PSStyle.OutputRendering` to `Ansi` (Internal 18394)
- Update `HelpInfoUri` for 7.2 release (#16456)
- Fix typo for "privacy" in MSI installer (#16407)

### Tests

- Set clean state before testing `UseMU` in the MSI (#16543)

### Build and Packaging Improvements

<details>

<ul>
<li>Add explicit job name for approval tasks in Snap stage (#16579)</li>
<li>Fixing the build by removing duplicate TSAUpload entries (Internal 18399)</li>
<li>Port CGManifest fixes (Internal 18402)</li>
<li>Update CGManifest (Internal 18403)</li>
<li>Updated package dependencies for 7.2.1 (Internal 18388)</li>
<li>Use different containers for different branches (#16434)</li>
<li>Use notice task to generate license assuming CGManifest contains all components (#16340)</li>
<li>Create compliance build (#16286)</li>
<li>Update release instructions with link to new build (#16419)</li>
<li>Add diagnostics used to take corrective action when releasing buildInfoJson (#16404)</li>
<li>vPack release should use buildInfoJson new to 7.2 (#16402)</li>
<li>Add checkout to build json stage to get <code>ci.psm1</code> (#16399)</li>
<li>Update the usage of metadata.json for getting LTS information (#16381)</li>
<li>Move mapping file into product repo and add Debian 11 (#16316)</li>
</ul>

</details>

[7.2.1]: https://github.com/PowerShell/PowerShell/compare/v7.2.0...v7.2.1

### SHA256 Hashes of the release artifacts

- hashes.sha256
  - 7C6ABAF4BCB09959A7B908E25D05B8CFE0D55322C72765EB45CEB0650CFA2AD1
- powershell_7.2.1-1.deb_amd64.deb
  - 11756A6265BDC1EBD14D678AB08A6999C03E24157B26B2C2E40640E44CB46ACD
- powershell-7.2.1-1.rh.x86_64.rpm
  - 3A6516CE32B825038A207A2E2C4EE1B9BCE677EFE4BA220743CB46DE158BA421
- powershell-7.2.1-linux-alpine-x64.tar.gz
  - 4E39219B05567F1EB12B762371031D139181D43445D97410457420DB0BEF248A
- powershell-7.2.1-linux-arm32.tar.gz
  - D40CA2C8E05DD8B6169D7F35683432C671A3C58F35E46DEAF1D776D9A78B38FE
- powershell-7.2.1-linux-arm64.tar.gz
  - F0D6C9C36D69E1466E5A9412085EF52CAFD10B73F862D29479B806279A2975F4
- powershell-7.2.1-linux-x64.tar.gz
  - 337D9864799AD09B46D261071B9F835F69F078814409BC2681F4CC2857B6BDA5
- powershell-7.2.1-linux-x64-fxdependent.tar.gz
  - EEE9CF3DA1506AFF9AD53C71968E744FD630A2148F682D50ADB45370FF43E1EA
- powershell-7.2.1-osx-arm64.pkg
  - 034BDB6CA923036A4D4E2A065810F9CF96B07EA5781BC1C81330417433DD1EA6
- powershell-7.2.1-osx-arm64.tar.gz
  - 7478D10FD251CF41950392C5C357AB06E6CE439BC85E1C556EF125750957EC7C
- powershell-7.2.1-osx-x64.pkg
  - 23046DBADA76832E441B838170E3E36ADA54A69C4F3AE25E210BE554E659FCE4
- powershell-7.2.1-osx-x64.tar.gz
  - 331307ECF7CC28E7ECC7AFA7D0BA4527677996FDDFC0C08B913BB4B8ADCD8C3D
- PowerShell-7.2.1-win-arm32.zip
  - E49ADE364BB2D1818393C6D9D49036329FAD5061E2C7C9ACD6FC344575218D1B
- PowerShell-7.2.1-win-arm64.zip
  - 22EB1737DB326CB51C1ACD208300E61C80933DD8CBE11E8708122948413BB034
- PowerShell-7.2.1-win-fxdependent.zip
  - E9D21CEFEF18F336D949F136C1E8FEC16096D360CA4A1576203DC645C5CFEF53
- PowerShell-7.2.1-win-fxdependentWinDesktop.zip
  - EBE1185800EF318442C3C581FC405B61FFF9D9ED1AE009D87C93720911983A03
- PowerShell-7.2.1-win-x64.msi
  - 4BE52CDD6323E4BEBDABD39A1BF65CDFB78953F77FEDE957A8421C0F5F9B2676
- PowerShell-7.2.1-win-x64.zip
  - D08AAE1CBF53CA3ECA6EFCCB03B00178960BED6DAC5D151E83BB67A187C7E097
- PowerShell-7.2.1-win-x86.msi
  - 8E23111B45230E61EB489D9A6DB20154E66E4C73489E7F201C52E0C5738F6694
- PowerShell-7.2.1-win-x86.zip
  - A8916E7C35D4675FB09ADE0AB7161A4436606838513DD68D3C80D73CC3F7F4B5
- powershell-lts_7.2.1-1.deb_amd64.deb
  - C203B2A5770AEA2879E64D542DE87DC6A705988F8AAD5BAE6A759F03EAB6FE96
- powershell-lts-7.2.1-1.rh.x86_64.rpm
  - 3218FD5ABEBCD6549696D87AC53E23B246FC4E1C4AA1EDDBEA592FEAA7020CFB
- powershell-lts-7.2.1-osx-arm64.pkg
  - 22B2A11FF5344696B6E039BC88462C3E9F673F9AB8ECBFAB8636120FF4FF86C4
- powershell-lts-7.2.1-osx-x64.pkg
  - 2FD219A31DA51E1178BCB9E460B9CF05FDE3D9FD91E4C3CC0D63AE4AD4ED09F5

<hr /><em>This discussion was created from the release <a href='https://github.com/PowerShell/PowerShell/releases/tag/v7.2.1'>v7.2.1 Release of PowerShell</a>.</em>

**Comment by Tshegofatso-kubyana** [comment]
guys i have a problem with running a terminal in intelli j idea for java pp

**Reply by katnimmons** [reply]
what is the latest windows powershell for windows 11

**Comment by katnimmons** [comment]
don't understand ur language

On the go email!
katnimmons.com
8642072111
________________________________

**Comment by katnimmons** [comment]
Can’t seem to find it.



If this email contains an attachment, please reply as received.
***@***.***
***@***.***
[facebook]<https://www.facebook.com/kanimmons>
[linkedin]<https://www.linkedin.com/in/katnimmons/>
[instagram]<https://www.instagram/kanimmons>
Kat Nimmons

BROKER- REALTOR®

SRES® SFR® | RE/MAX® Executive

***@***.***
864-622-3642<tel:864-622-3642> | 864-207-2111<tel:864-207-2111>
***@***.***
***@***.******@***.***>
***@***.***
https://www.KatNimmons.com<https://www.katnimmons.com/>
***@***.***
100-A Miracle Mile Dr., Anderson, SC 29621


This email does not create a binding contract on our party unless the written agreement is signed in ink or electronically signed by our party and is attached as an image or PDF of the paper/electronic agreements.

**Comment by Olof-Lagerkvist** [comment]
Windows Defender detects "TrojanDownloader:MSIL/Tnega.ARP!MTB" in powershell-7.2.1-linux-x64-fxdependent.tar.gz

**Comment by javika90** [comment]
no entiendo como instalar la version mas nueva de power sshell

**Comment by yasinmillers** [comment]
how can i install the latest PowerShell

**Reply by doctordns** [reply]
There are several ways. You can use Chocolately, or the MS Store. OR, as I do, use the command
```powershell
iex "& { $(irm https://aka.ms/install-powershell.ps1) } -UseMSI"
```

**Comment by katnimmons** [comment]
Thanks Thomas

On Wed, Feb 23, 2022 at 7:42 AM Thomas Lee ***@***.***> wrote:

> There are several ways. You can use Chocolately, or the MS Store. OR, as I
> do, use the command
>
> iex "& { $(irm https://aka.ms/install-powershell.ps1) } -UseMSI"
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/PowerShell/PowerShell/discussions/16614#discussioncomment-2235978>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANNXLJQTK6HLIUP7YCD7QKLU4TI3HANCNFSM5KBSRWPQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 

Kat Nimmons

**Comment by deanland8** [comment]
Thanks Thomas
